debezium-core/src/main/java/io/debezium/schema/TopicSelector.java

```diff
@@ -82,4 +88,77 @@ public String getHeartbeatTopic() {
     public interface DataCollectionTopicNamer<I extends DataCollectionId> {
         String topicNameFor(I id, String prefix, String delimiter);
     }
+
```

Comment:
@gunnarmorling Is not here overlap with `io.debezium.util.SchemaNameAdjuster`? Should not they use the same code?

Reply:
I was thinking about that, too, but schema name rules are a bit more restrictive (regarding the first letter of the name).